### PR TITLE
Add left biphone recipe for swbd

### PR DESCRIPTION
**Pull request by naxingyu**
This is a WIP PR to track the progress in left biphone chain modeling, as requested by #766  . The results will be updated in the script instead of RESULT because it is WIP.

### CODE REVIEW

File: egs/swbd/s5c/local/chain/run_tdnn_7d.sh [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=12
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/tdnn_7b  # Note: _sp will get added to this if $speed_perturb == true.
```
Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/run_tdnn_7d.sh]: tdnn_7b --> tdnn_7d

File: egs/swbd/s5c/local/chain/run_tdnn_7d.sh [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  cp -r data/lang $lang
+  silphonelist=$(cat $lang/phones/silence.csl) || exit 1;
+  nonsilphonelist=$(cat $lang/phones/nonsilence.csl) || exit 1;
+  # Use our special topology... note that later on may have to tune this
+  # topology.
+  steps/nnet3/chain/gen_topo.py $nonsilphonelist $silphonelist >$lang/topo
+fi
+
+if [ $stage -le 11 ]; then
+  # Build a tree using our new topology.
```
Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/run_tdnn_7d.sh]: Add a comment here saying this is the critically different step compared to the other recipes.

File: egs/swbd/s5c/local/chain/run_tdnn_7d.sh [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+
+
+if [ $stage -le 13 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+    utils/create_split_dir.pl \
+     /export/b0{5,6,7,8}/$USER/kaldi-data/egs/swbd-$(date +'%m_%d_%H_%M')/s5c/$dir/egs/storage $dir/egs/storage
+  fi
+
+ touch $dir/egs/.nodelete # keep egs around when that run dies.
```
Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/run_tdnn_7d.sh]: Don't do this. If you want to keep the egs do `remove_egs=false` .

**Comment by vijayaditya** [comment]
I will merge once the conflicts have been resolved.

**Comment by naxingyu** [comment]
Thanks Vij. There should be no conflict. Maybe it is the WIP tag that 
disable the merging.
The '.nodelete' line of script came from run_tdnn_7b.sh. I'll deleted 
it, but I think there should be a cleanup in the swbd chain recipe dir, 
since it is quite noisy AS IS.

在 2016/5/13 10:24, Vijayaditya Peddinti 写道:

**Comment by naxingyu** [comment]
Updated and WIP removed. Ready to merge.

在 2016/5/13 10:24, Vijayaditya Peddinti 写道:

**Comment by vijayaditya** [comment]
Sorry @naxingyu I failed to merge this last time you cleared the conflicts. Could you do a rebase your branch as it is currently out-of-date ? I will merge immediately now.

**Comment by naxingyu** [comment]
I could, but it's strange that I don't see a conflict message on 
Github... The files I committed are not conflicting with any other PR.

在 2016/5/16 10:02, Vijayaditya Peddinti 写道:

**Comment by vijayaditya** [comment]
There are no conflicts, github is just saying that your branch is stale
compared to the master. A rebase would solve this issue.

--Vijay

**Comment by naxingyu** [comment]
I see. Is it ok now?

Xingyu

在 2016/5/16 10:18, Vijayaditya Peddinti 写道:

**Comment by danpovey** [comment]
I think it's OK.  It's  because yenda made that github configuration
change.  As long as it's only changes in scripts, it's always safe to merge
such things.

**Comment by vijayaditya** [comment]
Yep. Thanks for making the change. I merged the PR.

**Comment by danpovey** [comment]
@naxingyu , I notice in the RESULTS file in swbd/s5c there is a very out-of-date, suboptimal chain system listed there.  Currently I think this 7d is the recommended system, but I don't have access to it because you ran it.  Can you please remove the current chain+tdnn system in the RESULTS file and replace it with the corresponding results from this system?

**Comment by naxingyu** [comment]
OK


在 2016/8/5 8:04, Daniel Povey 写道:

**Pull request merged**